docs/concepts/what-is-ipfs.md

```diff
@@ -1,89 +1,90 @@
+- [Centralization](#centralization)
+- [Performance](#performance)
+- [Scalability](#scalablity)
+- [Link rot](#link-rot)
+- [Security](#security)
+- [Data sovereignty and ownership](#data-sovereignty)
+- [Verifiable off-chain storage](#verifiable-off-chain-storage)
+- [Local-first software](#local-first-software)
+- [Vendor lock-in](#vendor-lock-in)
 
-That last point is actually where IPFS gets its full name: the **InterPlanetary File System**. We're striving to build a system that works across places as disconnected or as far apart as planets. While that's an idealistic goal, it keeps us working and thinking hard, and almost everything we create in pursuit of that goal is also useful here at home.
+### Centralization
 
-## Content addressing
+IPFS is an open-source, decentralized system that eliminates the need for a central authority, making it more resilient and censorship-resistant than traditional centralized systems. No single entity or person controls, manages or owns IPFS; rather, it is a community-maintained project with multiple implementations of the protocol, multiple tools and apps leveraging that protocol, and multiple users and organizations contributing to it's design and development.
 
-::: callout
-For a beginner-friendly primer on why cryptographic hashing and content addressing matter, take a look at ProtoSchool's tutorial, [Content Addressing on the Decentralized Web](https://proto.school/content-addressing).
-:::
+### Performance
 
-What about that link to the aardvark page above? It looked a little unusual:
+IPFS provides faster access to data by storing it at multiple locations, and allowing users to access it from the nearest location using content addressing instead of <VueCustomTooltip label="Data identified and linked to by it's location. An example is HTTP." underlined>location-based addressing</VueCustomTooltip>. In other words, because data can be addressed based on it's contents, a node on the network can fetch that data from _any_ other node in the netork that has the data; thus, performance issues like latency are reduced. 
 
-```
-/ipfs/bafybeiaysi4s6lnjev27ln5icwm6tueaw2vdykrtjkwiphwekaywqhcjze/wiki/Aardvark
-```
+### Scalability
 
-That jumble of letters after `/ipfs/` is called a [_content identifier_](content-addressing.md) and it’s how IPFS can get content from multiple places.
+IPFS has no single point of failure, and users do not need to trust each other. In other words, the failure of a single or even multiple nodes in the network does not affect the functioning of the entire network, and 
 
-Traditional URLs and file paths such as…
+### Link rot
```

Comment:
This one feels a little weak. This one is an important point but could be more refined.